configure.ac

```diff
@@ -594,6 +594,20 @@ AS_IF([test "$enable_ndb" = yes],[
 ])
 AM_CONDITIONAL([NDB], [test "$enable_ndb" = yes])
 
+#=================
+# Process --enable-bdb-ro
+AC_ARG_ENABLE([bdb-ro], [AS_HELP_STRING([--enable-ndb-ro (EXPERIMENTAL)],[enable the read-only Berkeley DB code])],
```

Comment:
`--enable-ndb-ro` -> `--enable-bdb-ro`

Freudian slip? ;)

Reply:
No, copy'n'pasted from the ndb lines.